app/resources/messages.ts

```diff
@@ -22,6 +22,20 @@ class Messages {
 		\nUnless you have a good reason to believe otherwise, you should not proceed.
 		\nYou can click here if you'd like to proceed with the connection.`;
 	}
+
+	enterpriseOrgError(length: number): any {
```

Comment:
@kanishk98 You can create an interface here instead of any as you already know that it has title and content only. 
```
interface ErrorType {
    title: string;
    content: string;
}
```
and then use it here, instead of any. Same for orgRemovalError.